pkg/debug/scripts/collect-info.sh

```diff
@@ -5,7 +5,7 @@
 #
 
 # Script version, don't forget to bump up once something is changed
-VERSION=13
+VERSION=14
```

Comment:
The current version in the master is 15, so it should be switched to 16.

Reply:
Thanks for pointing it out. Will update.